OpenRA.Game/FileFormats/AudLoader.cs

```diff
@@ -170,11 +170,15 @@ public static bool LoadSound(Stream s, out byte[] rawData, out int sampleRate)
 			out int sampleRate)
 		{
 			channels = sampleBits = sampleRate = 0;
+			var position = stream.Position;
 
 			try
 			{
 				if (!LoadSound(stream, out rawData, out sampleRate))
+				{
+					stream.Seek(position, SeekOrigin.Begin);
```

Comment:
Please use `stream.Position = position` instead of `stream.Seek` everywhere for consistency with our other file parsers.